.travis.yml

```diff
@@ -16,16 +16,17 @@ matrix:
 
 before_install:
     - CI_ENV=`bash <(curl -s https://codecov.io/env)`
-    - travis_retry docker pull nuto/nuto_docker:xenial
-    - docker run $CI_ENV -itd --user nuto --name dock -v $(pwd):/home/nuto/source nuto/nuto_docker:xenial
+    - travis_retry docker pull nuto/nuto_docker:dev
```

Comment:
The name is fine for now, I guess. Do we want to establish a naming convention?

Reply:
I thought that maybe a Docker image that has NuTo installed will follow in the future (see #159), and since this is one that has all the dependencies installed in case you want to start developing NuTo, not necessarily using it, we'd call this one `dev` and the other maybe `run` or `compute` or something.